docarray/array/storage/base/backend.py

```diff
@@ -77,7 +78,14 @@ def _map_embedding(self, embedding: 'ArrayType') -> 'ArrayType':
     def _map_type(self, col_type: str) -> str:
         return self.TYPE_MAP[col_type].type
 
-    def _normalize_columns(self, columns):
+    def _normalize_columns(
+        self, columns: Optional[Union[List[Tuple[str, str]], Dict[str, str]]]
+    ) -> Dict[str, str]:
         if columns is None:
-            return []
+            return {}
+        if isinstance(columns, list):
+            warnings.warn(
+                'Using "columns" as a List of Tuples will be deprecated soon. Please provide a Dictionary'
+            )
+            columns = {col_desc[0]: col_desc[1] for col_desc in columns}
```

Comment:
```suggestion
            columns = {col: col_type for col, col_type in columns}
```

Reply:
Like this we can cover in the case a List[List] is passed